xija/__init__.py

```diff
@@ -3,7 +3,7 @@
 from .component import *
 from .files import files
 
-__version__ = '4.15'
+__version__ = '4.16'
```

Comment:
We're moving to SCM versioning in pretty short order (before this PR gets released) so you might as well just revert this change otherwise it will generate a merge conflict.